packages/table/src/locator.ts

```diff
@@ -24,12 +24,26 @@ export interface ILocator {
     convertPointToColumn: (clientX: number) => number;
 
     /**
+     * Locates a column boundary given the client X coordinate. The boundary
+     * is returned as the index of the column appearing to the right of that
+     * boundary. Returns -1 if the coordinate is not over a column.
+     */
+    convertPointToColumnLeftBoundary: (clientX: number) => number;
```

Comment:
@giladgray That naming would be ambiguous unfortunately. See this description of the already existing `convertPointToColumn`:

```
/**
 * Locates a column's index given the client X coordinate. Returns -1 if
 * the coordinate is not over a column.
 */
convertPointToColumn: (clientX: number) => number;
```

`convertPointToColumnLeftBoundary` is simply adding support for an extra column index off to the right (use case: if you're reordering and mousing over the right half of column `N`, this function would return the next greater index, such that a vertical guide would then show on the left boundary of column `N+1`).

Maybe it's better to just add an extra parameter to `convertPointToColumn`? Such a parameter would let us add support for an extra column on the end, rather than returning -1.